### PR TITLE
Add README details about re: broccoli-asset-rev

### PR DESCRIPTION
**Pull request by lukemelia**
Added a section about working with broccoli-asset-rev and some other clarifications to make things clearer that confused me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The `ManifestGenerator` will generate an asset manifest and merge it into your b
 generates the manifest according to the options specified in `manifestOptions`:
 
 * The `bundlesLocation` option is a string that specifies which directory in the build tree contains the bundles to be
-placed into the asset manifest. This defaults `bundles`.
+placed into the asset manifest. This defaults to `bundles`. Each bundle is a directory containing files that will be be downloaded when the bundle is requested. You are responsible for getting the right files into those directories.
```
Comment (trentmwillis) [inline README.md]: Repeated word: `be be`.

File: README.md [hunk 2]
```diff
@@ -43,8 +43,9 @@ _Note: This class provides default `contentFor`, `postprocessTree`, and `postBui
 
 ### Why isn't a manifest generated by default?
 
-No manifest is generated by default since there is no convention for bundling assets within Ember currently. Thus, to
-prevent introducing unintuitive or conflicting behavior, we provide no default generation.
+This addon doesn't perform manifest generator just by virtue of being installed because there is no convention for
```
Comment (trentmwillis) [inline README.md]: Small grammar thing: `generator` should be `generation`

File: README.md [hunk 3]
```diff
@@ -76,6 +77,28 @@ var app = new EmberApp(defaults, {
 
 The function receives the `filePath` for each asset and must return a string.
 
+### Using With `broccoli-asset-rev`
```
Comment (trentmwillis) [inline README.md:80]: I'm unsure we need an example this long. I think we simply need to mention that `broccoli-asset-rev` needs to run after your addon and that you can specify that via the `ember-addon.after` field in the `package.json`.

I want to make sure the readme doesn't become overly verbose (makes it harder to locate things quickly). If you feel this much detail is definitely needed, maybe we could start up a wiki or additional docs folder.

**Review by trentmwillis — CHANGES_REQUESTED** [review]
Thanks for putting this together!

A few smallish changes I'd like to see, but overall this is looking solid.

File: README.md (hunk 3 above)
Comment (lukemelia) [reply README.md:80]: Given that `broccoli-asset-rev` ships with ember-cli apps by default, I feel strongly that this belongs in the README in step-by-step fashion unless we can make the steps unnecessary. This was the biggest stumbling block for me in trying to adopt this addon after seeing your talk.

**Comment by lukemelia** [comment]
I'm sorry this sat for so long -- I missed the notification of your review @trentmwillis and just saw it while returning to the github repo for another reason.

File: README.md (hunk 3 above)
Comment (trentmwillis) [reply README.md:80]: Okay, seems fine then. I'm not super against and as someone who is super close to the project, I am likely less attuned to what the readers need.

**Review by trentmwillis — APPROVED** [review]

**Pull request merged**